app/Connect/Support/GeneratesLegacyAuditComment.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Connect\Support;
+
+use App\Models\Comment;
+
+trait GeneratesLegacyAuditComment
+{
+    protected function addLegacyAuditComment(int $articleType, int $articleId, string $comment): void
+    {
+        $comment = Comment::create([
+            'ArticleType' => $articleType,
+            'ArticleID' => $articleId,
+            'user_id' => Comment::SYSTEM_USER_ID,
+            'Payload' => $comment,
+        ]);
```

Comment:
nit: `string $comment` is immediately overwritten by this created `Comment` model. It works, but it's kinda confusing. I recommend renaming the `string $comment` arg to something like `$payload`.

Reply:
I could easily be confused by that myself. Updated.